install-info uses symbols from zlib so must link against it.

Upstream-Status: Pending

Signed-off-by: Joshua Lock <josh@linux.intel.com>

Index: texinfo-5.1/install-info/Makefile.am
===================================================================
--- texinfo-5.1.orig/install-info/Makefile.am
+++ texinfo-5.1/install-info/Makefile.am
@@ -34,4 +34,4 @@ AM_CPPFLAGS = 					\
   -I$(top_srcdir)/gnulib/lib			\
   -I$(top_builddir)/gnulib/lib			\
   -DLOCALEDIR=\"$(localedir)\"
-LDADD = $(top_builddir)/gnulib/lib/libgnu.a $(LIBINTL)
+LDADD = $(top_builddir)/gnulib/lib/libgnu.a $(LIBINTL) -lz
